Formula/php@7.2.rb

```diff
@@ -2,9 +2,8 @@ class PhpAT72 < Formula
   desc "General-purpose scripting language"
   homepage "https://www.php.net/"
   # Should only be updated if the new version is announced on the homepage, https://www.php.net/
-  url "https://www.php.net/distributions/php-7.2.33.tar.xz"
-  mirror "https://fossies.org/linux/www/php-7.2.33.tar.xz"
```

Comment:
We should keep the mirror.

Reply:
You're right. I have no idea why that got removed. Fixed now. Thanks!